packages/api/src/repository-infos.ts

```diff
@@ -0,0 +1,30 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { fromUrl } from 'hosted-git-info';
+
+import rootPackage from '../../../package.json' with { type: 'json' };
+
+const infos = fromUrl(rootPackage.repository);
+
+if (!infos) {
+  throw new Error(`Could not parse repository information from URL: ${rootPackage.repository}`);
+}
+
+export const ORGANIZATION_NAME = infos.user;
+export const REPOSITORY_NAME = infos.project;
```

Comment:
after some thinking, I think it should be named GITHUB_OWNER and GITHUB_REPOSITORY

let say we move to GitLab for example using Octokit and passing constants won't work anyway so it's really specific to GitHub

there can be a check that if the infos.type is not GitHub it's also failing

Reply:
not blocker: you could also expose REPOSITORY_URL which would be the content of rootPackage.repository and rootPackage.homepage

(because I see some files in the repository using the import of package.json and instead they could import these constants)